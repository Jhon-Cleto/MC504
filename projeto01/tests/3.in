2 2
2 2
4321