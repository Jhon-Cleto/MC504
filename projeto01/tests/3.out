Multiplicação de Matrizes: A x B = C

Matriz A (2 x 2):
 004 004
 007 001

Matriz B (2 x 2):
 001 008
 002 008

Matriz C (2 x 2):
 012 064
 009 064
